src/sentiment.py

```diff
@@ -0,0 +1,302 @@
+# Code adapted from https://github.com/Himabindugssn/Sentiment-classification-using-transformers
```

Comment:
This code is awesome! Nice job.

Reply:
Thanks... i mean i can't really take too much credit for it!